### ISSUE TITLE
Make links more accessible

### ISSUE THREAD

**Issue by LukasKalbertodt** [elan-ev/tobira#799]
Our links in text blocks and similar text content do not have enough contrast *with the other text*. It should be easy to see all links at a glance. So what's needed is a "contrast triangle" where the background, normal text and links need to have sufficient contrast to each other. Also see: https://contrast-triangle.com/

This can either be done by choosing another color or by adding contrast in another way, e.g. by underlining the links. I think in our case, underlining might be the better option as we don't have that much control over the color. And I don't really think it looks bad.

To be clear: I am just talking about links within other text. Not the nav box, not video grid items, and so on.
